CHANGELOG.rst

```diff
@@ -7,6 +7,7 @@ releases, in reverse chronological order.
 v2.2.0
 ------
 
+* Added the Granular priority support
```

Comment:
This isn't very informative, something like "add support for filtering by priority"